dependencies/vaticle/repositories.bzl

```diff
@@ -38,6 +38,7 @@ def vaticle_typedb_common():
 def vaticle_typedb_behaviour():
     git_repository(
         name = "vaticle_typedb_behaviour",
-        remote = "https://github.com/vaticle/typedb-behaviour",
-        commit = "767bf98fef7383addf42a1ae6e97a44874bb4f0b" # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_behaviour
+        remote = "https://github.com/shiladitya-mukherjee/typedb-behaviour",
+        commit = "ac3006755c13e40bb1404f82ab9e65d4255e3cce" # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_behaviour
```

Comment:
Since typedb-behaviour is now merged, You can now bump this to the new vaticle commit.